### ISSUE TITLE
Remove [role=presentation] on the Pager template

### ISSUE THREAD

**Issue by sarongebre** [unlcms/unl_five#86]
Fails the aXe Accessibility Metric issues on Web Audit. 

This issue was first created on CMS main project  https://github.com/unlcms/project-herbie/issues/972
